docs/manual/scala/guide/cluster/PersistentEntity.md

```diff
@@ -20,19 +20,21 @@ When an entity is started it replays the stored events to restore the current st
 
 ## Choosing a database
 
-Lagom supports the following databases:
+Lagom is compatible with the following databases:
 
 * [Cassandra](https://cassandra.apache.org/)
 * [PostgreSQL](https://www.postgresql.org/)
 * [MySQL](https://www.mysql.com/)
 * [Oracle](https://www.oracle.com/database/index.html)
 * [H2](https://www.h2database.com/)
+* [Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/)
+* [Couchbase](https://www.couchbase.com/)
 
-We recommend using Cassandra. Cassandra is a very scalable distributed database, and it is also flexible enough to support typical use cases of reactive services. In contrast to most relational databases, it natively supports sharding and replication, and is emerging as an industry standard open source NoSQL database.
+For instructions on configuring your project to use Cassandra, see [[Using Cassandra for Persistent Entities|PersistentEntityCassandra]]. If instead you want to use one of the relational databases listed above, see [[Using a Relational Database for Persistent Entities|PersistentEntityRDBMS]] on how to configure your project. If you wish to use Couchbase, proceed to the [Lagom section of the plugin site](https://doc.akka.io/docs/akka-persistence-couchbase/current/lagom-persistent-entity.html) for all the details.
 
-Lagom also provides out of the box support for running Cassandra in a development environment - developers do not need to install, configure or manage Cassandra at all themselves when using Lagom, which makes for great developer velocity, and it means gone are the days where developers spend days setting up their development environment before they can start to be productive on a project.
+To see how to combine Cassandra for write-side persistence and JPA for a read-side view, see the [Mixed Persistence Service](https://github.com/lagom/lagom-recipes/blob/master/mixed-persistence/mixed-persistence-scala-sbt/README.md) recipe.
```

Comment:
The Scala example uses plain JDBC instead of JPA for the read side.

Reply:
My bad, copy pasted the section from the Java writeup.

Reply:
Fixed the text and the link (the link in the Java markdown pointed to the scala recipe)